upgrading/os_upgrades.adoc

```diff
@@ -31,11 +31,19 @@ Use the following to safely upgrade the OS on a host:
 $ oc adm drain <node_name> --force --delete-local-data --ignore-daemonsets
 ----
 
-. Update the host packages, and reboot the host. A reboot ensures that the host is
+. Make sure all the sensitive packages are protected by exclude rules, update
```

Comment:
Please add a separate step to enable the excluder rules. I'd restore line 34 to its original state and keep the update and reboot commands in this step.

In the new step, please give a bit more context about why you'd want to do enable these rules.